[2023-10-19T13:24:37.281+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_notification.send_message_to_slack manual__2023-10-19T13:24:21.354601+00:00 [queued]>
[2023-10-19T13:24:37.296+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_notification.send_message_to_slack manual__2023-10-19T13:24:21.354601+00:00 [queued]>
[2023-10-19T13:24:37.297+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-10-19T13:24:37.307+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): send_message_to_slack> on 2023-10-19 13:24:21.354601+00:00
[2023-10-19T13:24:37.323+0000] {standard_task_runner.py:57} INFO - Started process 3064 to run task
[2023-10-19T13:24:37.339+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'stock_notification', 'send_message_to_slack', 'manual__2023-10-19T13:24:21.354601+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/stock_notification.py', '--cfg-path', '/tmp/tmp2oz2i0a3']
[2023-10-19T13:24:37.342+0000] {standard_task_runner.py:85} INFO - Job 46: Subtask send_message_to_slack
[2023-10-19T13:24:37.369+0000] {task_command.py:416} INFO - Running <TaskInstance: stock_notification.send_message_to_slack manual__2023-10-19T13:24:21.354601+00:00 [running]> on host 46830eaaaf4f
[2023-10-19T13:24:37.675+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='joshua881117@gmail.com' AIRFLOW_CTX_DAG_OWNER='Joshua Lin' AIRFLOW_CTX_DAG_ID='stock_notification' AIRFLOW_CTX_TASK_ID='send_message_to_slack' AIRFLOW_CTX_EXECUTION_DATE='2023-10-19T13:24:21.354601+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-19T13:24:21.354601+00:00'
[2023-10-19T13:24:37.683+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/slack_sdk/web/internal_utils.py:416: UserWarning: client.files_upload() may cause some issues like timeouts for relatively large files. Our latest recommendation is to use client.files_upload_v2(), which is mostly compatible and much stabler, instead.
  warnings.warn(message)

[2023-10-19T13:24:38.333+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-19T13:24:38.361+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=stock_notification, task_id=send_message_to_slack, execution_date=20231019T132421, start_date=20231019T132437, end_date=20231019T132438
[2023-10-19T13:24:38.462+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T13:24:38.550+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
